eth-contracts/contracts/staking/Staking.sol

```diff
@@ -96,7 +95,7 @@ contract Staking is RegistryContract, ERCStaking, ERCStakingHistory, IsContract,
             bytes("")); // TODO: RM bytes requirement if unused
 
         // Update claim history even if no value claimed
-        accounts[_stakerAccount].claimHistory.add64(getBlockNumber64(), _amount);
+        accounts[_stakerAccount].claimHistory.add(block.number, _amount);
```

Comment:
why move from uint64 to uint192 math here?

Reply:
everything was already going to uint192 so this conversion was pointless, also removes our dependency on the vendored TimeHelpers.sol - https://github.com/AudiusProject/audius-protocol/blob/3a0ad8685be35d1d3171b46ed2d6e767c77a8ea7/eth-contracts/contracts/staking/Checkpointing.sol#L33